solutions/gold/usaco-862.mdx

```diff
@@ -0,0 +1,105 @@
+---
+id: usaco-862
+source: USACO Gold 2018 December
+title: Cowpatibility
+author: Ryan Chou
+---
+
+[Official Analysis (C++)](http://www.usaco.org/current/data/sol_cowpatibility_gold_dec18.html)
+
+<Spoiler title="Hint">
+
+Instead of trying to find the number of incompatible pairs, try to find the number of *compatible* pairs instead!
+
+This is known as [complementary counting](https://artofproblemsolving.com/wiki/index.php/Complementary_counting).
```

Comment:
```suggestion
Instead of trying to find the number of incompatible pairs, try to find the number of *compatible* pairs instead using [complementary counting](https://artofproblemsolving.com/wiki/index.php/Complementary_counting).
```

Reply:
i think it's better split up -- makes the sentence harder to read.